index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Survey form</title>
+
+  <link rel="stylesheet" href="mystyle.css">
+
+  <meta charset="UTF-8" name="viewport" content="width=device-width, initial-scale=1.0">
+  
+  
+</head>
+<body>
+<h1 id="title"> Survey Form </h1>
+<p id="description">
+  Collaboration work on survey form
+ </p>
+ <form id="survey-form">
+   <label for="name-label">Name:</label><br>
+   <input id="name" type="text" placeholder="input name"><br>
+   <label for="email-label">Email:</label><br>
+   <input id="email" type="email" placeholder="input email"><br>
+   <label for="number-label">Number:</label><br>
+   <input id="number" type="number" min="0" max="50" placeholder="no.">
+   <select id = "dropdown">
+      <option value = "choose one">computer</option>
+      <option value = "js">js</option>
+      <input id="textarea" type="text" placeholder="additional comments">
```

Comment:
For each of these elements, please use a "class" attribute to help identify and style in css. For example;
<input id="name" type="text" **class="name"**>

Reply:
- [ ] Great job :100: